src/encoded/static/components/matrix_target.js

```diff
@@ -0,0 +1,677 @@
+                this.setState({ scrolledRight: false });
+            }
+        } else if (!this.state.scrolledRight) {
+            this.setState({ scrolledRight: true });
+        }
+    }
+
+    render() {
+        const { context } = this.props;
+        const { scrolledRight, targetData, showOrganismRequest, selectedSubTab } = this.state;
+
+        const headers = [
+            {
+                organismName: 'Homo sapiens',
+                assayTitle: 'Histone ChIP-seq',
+                title: 'Homo sapiens | Histone ChIP-seq',
+                url: 'replicates.library.biosample.donor.organism.scientific_name=Homo sapiens&assay_title=Histone ChIP-seq',
+            }, {
+                organismName: 'Homo sapiens',
+                assayTitle: 'TF ChIP-seq',
+                title: 'Homo sapiens | TF ChIP-seq',
+                url: 'replicates.library.biosample.donor.organism.scientific_name=Homo sapiens&assay_title=TF ChIP-seq',
+            }, {
+                organismName: 'Mus musculus',
+                assayTitle: 'Histone ChIP-seq',
+                title: 'Mus musculus | Histone ChIP-seq',
+                url: 'replicates.library.biosample.donor.organism.scientific_name=Mus musculus&assay_title=Histone ChIP-seq',
+            }, {
+                organismName: 'Mus musculus',
+                assayTitle: 'TF ChIP-seq',
+                title: 'Mus musculus | TF ChIP-seq',
+                url: 'replicates.library.biosample.donor.organism.scientific_name=Mus musculus&assay_title=TF ChIP-seq',
+            },
+        ];
+
+        const fontColors = globals.biosampleTypeColors.colorList(this.targetMatrixData.subTabs, { merryGoRoundColors: true });
+
+        return (
+            <div className="matrix__presentation">
+                <Spinner isSpinnerActive={this.state.isSpinnerActive} />
+                <div className={`matrix__label matrix__label--horz${!scrolledRight ? ' horz-scroll' : ''}`}>
+                    <span>biosample</span>
+                    {svgIcon('largeArrow')}
+                </div>
+                <div className="matrix__presentation-content">
+                    <div className="matrix__label matrix__label--vert"><div>{svgIcon('largeArrow')}{context.matrix.y.label}</div></div>
+                    {showOrganismRequest ? <GetTargets /> : null }
+                    <TargetTabPanel tabs={headers} selectedTab={this.state.selectedTab} tabPanelCss="matrix__data-wrapper">
+                        <TargetTabPanel tabs={this.subTabsHeaders} selectedTab={selectedSubTab} tabPanelCss="matrix__data-wrapper" handleTabClick={this.subTabClicked} fontColors={fontColors}>
+                            {targetData ?
+                                  <div className="matrix__data" onScroll={this.handleOnScroll} ref={(element) => { this.scrollElement = element; }}>
+                                      <DataTable tableData={convertTargetDataToDataTable(targetData, selectedSubTab)}> test </DataTable>
```

Comment:
Removed TargetDataTable and using DataTable as requested. I am using targetData (extracted from context) a lot because I need data structure I can easily extract information from like for search, filtering, result count and a mixture of these. That is not so easy with context.